core/templates/dev/head/app.js

```diff
@@ -471,7 +471,8 @@ oppia.factory('oppiaDatetimeFormatter', ['$filter', function($filter) {
       var date = new Date(millisSinceEpoch);
       if (date.toLocaleDateString() === new Date().toLocaleDateString()) {
         // The replace function removes 'seconds' from the time returned.
-        return date.toLocaleTimeString().replace(/:\d\d /, ' ');
+        return date.toLocaleTimeString('en-US',
```

Comment:
Eh, 24-hour format is probably less "friendly" than 12-hour format. I think the browser differences are minor, and 11:13 am vs 11:13 AM is totally fine. So, feel free to go ahead with that!